src/EditorFeatures/Core/Implementation/SplitComment/AbstractCommentSplitter.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using static Microsoft.CodeAnalysis.Formatting.FormattingOptions;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.SplitComment
+{
+    internal abstract class AbstractCommentSplitter
+    {
+        protected Document _document;
+        protected int _cursorPosition;
+        protected SourceText _sourceText;
+        protected SyntaxNode _root;
+        protected int _tabSize;
+        protected bool _useTabs;
+        protected CancellationToken _cancellationToken;
+        protected SyntaxTrivia _trivia;
+        protected bool _hasSpaceAfterComment;
+        protected IndentStyle _indentStyle;
+
+        protected SyntaxNode GetNodeToReplace() => _trivia.SyntaxTree.GetRoot();
```

Comment:
what is going on here?  is this just getting the root of the whole tree?

Reply:
Fixed to only get the parent of the trivia's token